### PR TITLE
Added serverlist command (owner only)

### PR DESCRIPTION
**Pull request by iShauny**

### CODE REVIEW

File: Floofbot/Modules/Utilities.cs [hunk 1]
```diff
@@ -139,5 +141,60 @@ public async Task ServerInfo()
...
+            List<PaginatedMessage.Page> pages = new List<PaginatedMessage.Page>();
+
+            foreach (SocketGuild g in guilds)
+            {
+                List<EmbedFieldBuilder> fields = new List<EmbedFieldBuilder>();
+
+                fields.Add(new EmbedFieldBuilder()
+                {
+                    Name = $"Owner",
+                    Value = $"{g.Owner.Username}#{g.Owner.Discriminator} | ``{g.Owner.Id}``",
```
Comment (JasperGray) [inline Floofbot/Modules/Utilities.cs:159]: Sanity check: is this the server owner or the bot owner?
Maybe change the name field to "Server Owner"

File: Floofbot/Modules/Utilities.cs [hunk 2]
```diff
@@ -139,5 +141,60 @@ public async Task ServerInfo()
...
+                fields.Add(new EmbedFieldBuilder()
+                {
+                    Name = $"Members",
+                    Value = g.MemberCount,
+                    IsInline = false
+                });
+
+                pages.Add(new PaginatedMessage.Page
+                {
+                    Author = new EmbedAuthorBuilder { Name = g.Name },
```
Comment (JasperGray) [inline Floofbot/Modules/Utilities.cs:177]: Actually, what does Author here do?
Is it like the "Title" field?

File: Floofbot/Modules/Utilities.cs [hunk 3]
```diff
@@ -139,5 +141,60 @@ public async Task ServerInfo()
...
+                Options = PaginatedAppearanceOptions.Default,
+                TimeStamp = DateTimeOffset.UtcNow
+            };
+            await PagedReplyAsync(pager, new ReactionList
+            {
+                Forward = true,
+                Backward = true,
+                Jump = true,
+                Trash = true
+            }, true);
```
Comment (JasperGray) [inline Floofbot/Modules/Utilities.cs:197]: What's this true parameter for?

File: Floofbot/Modules/Utilities.cs [hunk 4]
```diff
@@ -1,14 +1,16 @@
...
+using Discord.Addons.Interactive;
 using Discord.Commands;
 using Discord.WebSocket;
 
 namespace Floofbot
 {
-    public class Utilities : ModuleBase<SocketCommandContext>
+    public class Utilities : InteractiveBase
     {
         private static readonly Discord.Color EMBED_COLOR = Color.Magenta;
```
Comment (JasperGray) [inline Floofbot/Modules/Utilities.cs:15]: We'll need to figure out these colours and where to place them in a future PR considering that the serverlist command has a dark green (admin) embed colour right now.

File: Floofbot/Modules/Utilities.cs (hunk 3 above)
Comment (iShauny) [reply Floofbot/Modules/Utilities.cs:197]: Only the person who ran the command will have their reactions counted

File: Floofbot/Modules/Utilities.cs (hunk 2 above)
Comment (iShauny) [reply Floofbot/Modules/Utilities.cs:177]: Kinda like the title per page

File: Floofbot/Modules/Utilities.cs (hunk 1 above)
Comment (iShauny) [reply Floofbot/Modules/Utilities.cs:159]: Server owner

File: Floofbot/Modules/Utilities.cs (hunk 4 above)
Comment (iShauny) [reply Floofbot/Modules/Utilities.cs:15]: Lol didn't even notice this I can swap it to the random colour?

**Pull request merged**